setup/module/variables.tf

```diff
@@ -16,8 +16,8 @@ variable "username" {}
 
 variable "password" {}
 
-variable "time_between_requests" {
-  default = "10ms"
+variable "request_delay" {
+  default = "1ms"
```

Comment:
Is this change intentional?

Reply:
yeah, didn't see any reason for it to be relatively slow by default.

Reply:
(wouldn't be opposed to using nanoseconds)